packages/css-library/config.js

```diff
@@ -1,6 +1,25 @@
+const { transform } = require('@divriots/style-dictionary-to-figma');
+
 module.exports = {
   source: ['tokens/**/*.json'],
+  format: {
+    figmaTokensPlugin: ({ dictionary }) => {
+      // currently Figma only supports syncing colors, so only transform dictionary.tokens.color
+      const transformedTokens = transform(dictionary.tokens.color, { cleanMeta: ['filePath', 'isSource', 'original', 'attributes', 'category', 'type', 'path'] });
```

Comment:
removed `defaultToken:false` as it caused the colors to not be labeled well:

Before:
![Screenshot 2023-08-17 at 4 12 34 PM](https://github.com/department-of-veterans-affairs/component-library/assets/1413938/25e08964-bdda-4aeb-807e-3b3b30c83b8f)

Now:
![Screenshot 2023-08-17 at 4 12 42 PM](https://github.com/department-of-veterans-affairs/component-library/assets/1413938/08020d81-83ea-43c2-8912-3bb3981d1493)

I don't think I can get rid of the trailing hyphen because those are the defaults and they're labeled with an asterisk...